lib/date.nit

```diff
@@ -0,0 +1,254 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date and Time manipulation
+# The class Date has the basic operations to use only the date data
```

Comment:
Maybe change that in a less heavy description:

~~~nit
# The date and time behavior is splited in 2 classes:
# * `Date` handles the year, month and day part of the date.
# * `Time` handles the time in hours, minutes and seconds.
# 
# These are extended by DateTime that merges the two concepts in one class.
~~~